web/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
is there a particular type error we can help with in order to maintain strict type checking?